contracts/GovernorAlpha.sol

```diff
@@ -3,13 +3,13 @@ pragma experimental ABIEncoderV2;
 
 contract GovernorAlpha {
     /// @notice The name of this contract
-    string public constant name = "Uniswap Governor Alpha";
+    string public constant name = "Vexchange Governor Alpha";
 
     /// @notice The number of votes in support of a proposal required in order for a quorum to be reached and for a vote to succeed
-    function quorumVotes() public pure returns (uint) { return 40_000_000e18; } // 4% of Uni
+    function quorumVotes() public pure returns (uint) { return 40_000_000e18; } // 4% of vex
 
     /// @notice The number of votes required in order for a voter to become a proposer
-    function proposalThreshold() public pure returns (uint) { return 10_000_000e18; } // 1% of Uni
+    function proposalThreshold() public pure returns (uint) { return 10_000_000e18; } // 1% of vex
```

Comment:
Capitalize `vex` for consistency across files?

Reply:
fixed